warehouse/templates/index.html

```diff
@@ -15,13 +15,18 @@
 
 {% block title %}PyPI - the Python Package Index{% endblock %}
 
-{% macro project_snippet(release) -%}
-<div class="package-snippet">
+{% macro project_snippet(release, class) -%}
+{% if class == "latest" %}
+<div class="package-snippet" data-target="show-latest-releases.latestRelease">
+{% else %}
+<div class="package-snippet" data-target="show-trending-projects.trendingProject">
```

Comment:
if you use a single controller, you can avoid this `if` completely and use something like `data-target="show-more.release"`

Reply:
I don't see how I can do this, `show-more.latestReleases` and `show-more.trendingProjects` are entirely different arrays, perhaps there is something clever I can do I don't think there is an explicit way I can do it